### PR TITLE
update docs to markdown

### PR DESCRIPTION
**Pull request by arlimus**
run 2/3
https://github.com/chef/inspec/issues/1119

### CODE REVIEW

**Comment by chef-supermarket** [comment]
Hi. I am an automated pull request bot named Curry. There are commits in this pull request whose authors are not yet authorized to contribute to Chef Software, Inc. projects or are using a non-GitHub verified email address. To become authorized to contribute, you will need to sign the Contributor License Agreement (CLA) as an individual or on behalf of your company. [You can read more on Chef's blog.](https://www.chef.io/blog/2014/06/23/changes-to-the-contributor-license-agreement-process)

## GitHub Users Who Are Not Authorized To Contribute

The following GitHub users do not appear to have signed a CLA:

* @arlimus

[Please sign the CLA here.](https://supermarket.chef.io/become-a-contributor)

File: docs/dsl_inspec.md [hunk 1]
```diff
@@ -0,0 +1,245 @@
...
+
+## Are PosgtreSQL passwords empty?
+
+The following test shows how to audit machines running PostgerSQL to ensure that passwords are not empty.
+
+```ruby
+control 'postgres-7' do
+  impact 1.0
+  title 'Don't allow empty passwords'
+  describe postgres_session('user', 'pass').query("SELECT * FROM pg_shadow WHERE passwd IS NULL;") do
```
Comment (chris-rock) [inline docs/dsl_inspec.md:98]: Should we introduce attributes here

File: docs/plugin_kitchen_inspec.html.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+To define a suite that pulls its run-list from the Chef Compliance server:
+
+    suites:
+      - name: compliance
+        run_list:
+          - recipe[ssh-hardening]
+        verifier:
+          inspec_tests:
+            - compliance://base/ssh
```
Comment (chris-rock) [inline docs/plugin_kitchen_inspec.html.md:22]: should we add a simpler example that does not require compliance server?

**Review by chris-rock — APPROVED** [review]

File: docs/dsl_inspec.md (hunk 1 above)
Comment (arlimus) [reply docs/dsl_inspec.md:98]: That's a great idea!!

File: docs/plugin_kitchen_inspec.html.md (hunk 2 above)
Comment (arlimus) [reply docs/plugin_kitchen_inspec.html.md:22]: Absolutely, thank you!!

**Pull request merged**